src/pages/assignments/components/assignment-card.vue

```diff
@@ -184,10 +167,21 @@ export default {
       return this.getValue(this.assignment, 'details', 'period_count')
     },
     endPhase () {
-      return this.startPhase && this.getEndPeriod({ value: this.startPhase.value, periodCount: this.periodCount })
+      return this.startPhase && this.getEndPeriod({ value: this.startPhase.value, periodCount: this.periodCount - 1 })
     },
     isExpired () {
-      return !this.startPhase || (this.endPhase && new Date(this.endPhase.endDate).getTime() < Date.now())
+      return !this.startPhase || (this.endPhase && this.endPhase.endDate && new Date(this.endPhase.endDate).getTime() < Date.now())
+    },
+    willExpireWithin15Days () {
```

Comment:
As discussed with Leonie, we are going to change this to '2 voting durations' instead of 15 days.